lib/init.c

```diff
@@ -55,7 +55,7 @@
 #ifdef _MSC_VER
 #include <errno.h>
 #define getlogin_r(a,b) ENXIO
-#define srandom srand
+//#define srandom srand
```

Comment:
Don't disable code by just commenting it out. Either delete it or #ifdef it out.

In this case, this works for earlier versions of VS so if you need to get rid of this define then you would need
to use an ifdef to ONLY remove it for VS2019 or later.

However, srand() has been part of the c standard since a very long time so if this fails to compile,  maybe you are missing some include you will need to bring in srand()